docs/source/markdown/podman-machine-inspect.1.md

```diff
@@ -46,6 +46,73 @@ Print usage statement.
 Inspect the specified Podman machine.
 ```
 $ podman machine inspect podman-machine-default
+[
+     {
+          "ConfigDir": {
+               "Path": "/Users/jacksparrow/.config/containers/podman/machine/applehv"
+          },
+          "ConnectionInfo": {
+               "PodmanSocket": {
+                    "Path": "/var/folders/9r/n3056v597wv2cq8s2j80bdnw0000gn/T/podman/podman-machine-default-api.sock"
+               },
+               "PodmanPipe": null
+          },
+          "Created": "2025-02-11T14:12:48.231836+05:30",
+          "LastUp": "2025-08-12T19:31:19.391294+05:30",
+          "Name": "podman-machine-default",
+          "Resources": {
+               "CPUs": 6,
+               "DiskSize": 100,
+               "Memory": 6144,
+               "USBs": []
+          },
+          "SSHConfig": {
+               "IdentityPath": "/Users/jacksparrow/.local/share/containers/podman/machine/machine",
+               "Port": 53298,
+               "RemoteUsername": "core"
+          },
+          "State": "running",
+          "UserModeNetworking": true,
+          "Rootful": false,
+          "Rosetta": true
+     }
+]
+```
+
+Show machine name and state:
+```
+$ podman machine inspect --format "{{.Name}}\t{{.State}}"
+podman-machine-default running
+```
+
+Show machine resource information:
+```
+$ podman machine inspect --format "Machine: {{.Name}}\nCPUs: {{.Resources.CPUs}}\nMemory: {{.Resources.Memory}} bytes\nDisk: {{.Resources.DiskSize}} bytes"
+Machine: podman-machine-default
+CPUs: 6
+Memory: 6144 bytes
+Disk: 100 bytes
+```
+
+Show machine configuration details:
+```
+$ podman machine inspect --format "{{.Name}}: {{.State}} (Rootful: {{.Rootful}}, User Networking: {{.UserModeNetworking}})"
+podman-machine-default: running (Rootful: false, User Networking: true)
+```
+
+Show machine uptime information:
+```
+$ podman machine inspect --format "Created: {{.Created}}\nLast Up: {{.LastUp}}\nState: {{.State}}"
+Created: 2025-02-11 14:12:48.231836 +0530 IST
```

Comment:
Spell check is flagging `IST`. When I tried `UTC` instead, it worked. I'm not sure if there's a way to handle `IST`.
```suggestion
Created: 2025-02-11 14:12:48.231836 +0530 UCT
```